client/state/global-sidebar/selectors.ts

```diff
@@ -56,8 +56,7 @@ export const getShouldShowCollapsedGlobalSidebar = (
 	sectionGroup: string,
 	sectionName: string
 ) => {
-	const isAllowedRegion =
-		sectionGroup === 'sites-dashboard' || sectionGroup === 'sites' || sectionName === 'plugins';
```

Comment:
I do wonder why was `sites` in here in the first place though, not sure if this might break something else?

Reply:
When clicking a site on `/sites`, the sidebar does not seem to collapse anymore 👀 

https://github.com/Automattic/wp-calypso/assets/5287479/6abf54b8-fbb4-44e9-a002-d9cea3571520

Reply:
apparently, `isAllowedRegion` was introduced in https://github.com/Automattic/wp-calypso/pull/90280

Reply:
> I do wonder why was sites in here in the first place though
> isAllowedRegion was introduced in https://github.com/Automattic/wp-calypso/pull/90280

Yes, I introduced `isAllowedRegion` because we were seeing the collapsed sidebar happen in the reader and other places. The sectionGroup for the overview pages seems to be 'sites' which is why it was included.

I think the issue is a few lines down below in the connecting logic somewhere:

```
isEnabled( 'layout/dotcom-nav-redesign-v2' ) &&
		isAllowedRegion &&
		( siteSelected ||
			siteLoaded ||
			( ! siteId && pluginsScheduledUpdatesEditMode ) ||
			isWithinBreakpoint( '<782px' ) )
```

Simply that that breakpoint check should probably be tied to a specific region ( maybe something like `( sectionGroup === 'sites-dashboard' && isWithinBreakpoint( '<782px' ) )` instead? 🤔 

If Im understanding correctly we want the collapsed sidebar when:
* We have a site selected on the dashboard 
* Are viewing site pane tabs (overview etc).
* Are simple on the site dashboard with a narrow <782px screen width.
    * I don't think this requirement of "unselected and collapsed" extends to any other areas outside `sites-dashboard` ?
* Note currently unsure about the pluginsScheduled case, but we can leave as is for now.


If thats all true, I think we can wrap that isWithinBreakpoint check to only apply to the sites-dashboard region.

I will test and push an update.